src/assets/stylesheets/_buttons.scss

```diff
@@ -125,6 +125,23 @@ button {
   }
 }
 
+.cc-button-grey {
+  background: $cc-color-bali-hai !important;
+  border: 1px solid transparent;
+  color: $cc-color-white;
+  &:focus, &:hover {
+    background: $cc-color-bali-hai !important;
+    color: $cc-color-white;
+  }
+  &.cc-button-large {
+    background: linear-gradient($cc-color-bali-hai, $cc-color-lochmara), $cc-color-lochmara;
+    &:focus, &:hover {
+      background: linear-gradient($cc-color-bali-hai, $cc-color-blue-lagoon);
```

Comment:
Are we using gradients on our react buttons?

cc: @developish, @redconfetti, @frankiepun, @nkutub